reactapplication/src/card.jsx

```diff
@@ -10,11 +15,16 @@ export default function Card({ testimonial, cardIndex, index }) {
         <img className="CardImage" {...testimonial.image} />
         <p className="reviewer">{testimonial.name}</p>
         <div className="">
+          <div className="rating">
+            {[...Array(rating)].map((e, i) => {
```

Comment:
`Array(rating)` le Array of length `rating` banaucha so, redundant bhayo yo step
Plus, rating bhanne props pass garnu pardiena hola

`testimonial.rating` garnu parla, `rating` bhanda ni